[Bug]: RC0 fails to run storybook with addon-onboarding

### Describe the bug

Error: Cannot find module './dist/manager.mjs'
Require stack:
- /Users/omni/git/bison-design-system/node_modules/.pnpm/@storybook+addon-onboarding@8.0.0-rc.0/node_modules/@storybook/addon-onboarding/preset.js

### To Reproduce

_No response_

### System

```bash
System:
    OS: macOS 13.5.1
    CPU: (10) arm64 Apple M1 Max
    Shell: 3.6.1 - /opt/homebrew/bin/fish
  Binaries:
    Node: 21.6.2 - /opt/homebrew/bin/node
    npm: 10.2.4 - /opt/homebrew/bin/npm
    pnpm: 8.10.2 - /opt/homebrew/bin/pnpm <----- active
  Browsers:
    Chrome: 122.0.6261.94
    Safari: 16.6
  npmPackages:
    @storybook/addon-essentials: 8.0.0-rc.0 => 8.0.0-rc.0 
    @storybook/addon-interactions: 8.0.0-rc.0 => 8.0.0-rc.0 
    @storybook/addon-links: 8.0.0-rc.0 => 8.0.0-rc.0 
    @storybook/addon-onboarding: 8.0.0-rc.0 => 8.0.0-rc.0 
    @storybook/blocks: 8.0.0-rc.0 => 8.0.0-rc.0 
    @storybook/react: 8.0.0-rc.0 => 8.0.0-rc.0 
    @storybook/react-vite: 8.0.0-rc.0 => 8.0.0-rc.0 
    @storybook/test: 8.0.0-rc.0 => 8.0.0-rc.0 
    eslint-plugin-storybook: ^0.6.15 => 0.6.15 
    storybook: ^8.0.0-rc.0 => 8.0.0-rc.0
```


### Additional context

The 8.0.0-rc0 addon-onboarding contains the following code:

```
function managerEntries(entry = []) {
  return [...entry, require.resolve('./dist/manager.mjs')]; // <-- errors here
}

module.exports = {
  managerEntries,
  ...require('./dist/preset'),
};
```
The dist contains `manager.js`, not `manage.mjs`

--- a/code/addons/onboarding/preset.js
+++ b/code/addons/onboarding/preset.js
@@ -1,5 +1,5 @@
 function managerEntries(entry = []) {
-  return [...entry, require.resolve('./dist/manager.mjs')];
+  return [...entry, require.resolve('./dist/manager.js')];
 }
 
 module.exports = {

